### PR TITLE
ALPACA is working..

### PR DESCRIPTION
**Pull request by tugberk92**

### CODE REVIEW

File: alpaca.sh [hunk 1]
```diff
@@ -12,6 +12,7 @@ env:
 #!/bin/sh
 rsync -a $SOURCEDIR/* .
 make
+cp -r * $INSTALLROOT
```
Comment (olantwin) [inline alpaca.sh]: It's quite possible that there is a `make install` target, which would avoid copying unnecessary files. Please use that one if at all possible!

**Comment by ThomasRuf** [comment]
May I ask:
 why Xerces.sh is being deleted, 
 why there is an blank line added to fairroot.sh
 why the change in line 46/47 of fairship.sh

Thomas

**Comment by tugberk92** [comment]
> why Xerces.sh is being deleted,

I was following your advice ([https://github.com/ShipSoft/shipdist/pull/34#issuecomment-562080478](url)), then I made a spelling mistake; it had to be xerces**c**.sh
So, basically, I deleted it. Was there any other action to take?

> why there is an blank line added to fairroot.sh

it has no use, let me reserve the blank

> why the change in line 46/47 of fairship.sh

the spacing is useless again, let me reverse it as well.

Thanks,
Atakan T.

File: alpaca.sh (hunk 1 above)
Comment (tugberk92) [reply alpaca.sh]: Yes, you suggested me this before. However, it says there is no rule to make it;

```make: *** No rule to make target `install'.  Stop.```

Comment (olantwin) [reply alpaca.sh]: In that case, I would still suggest only copying headers and binary files (alongside possibly generated documentation and other files you might need for the generation).

**Comment by olantwin** [comment]
> May I ask:
> why Xerces.sh is being deleted,
> why there is an blank line added to fairroot.sh
> why the change in line 46/47 of fairship.sh
> 
> Thomas

He added xerces.sh (not xerces**c**.sh) in his first commit and then deletes it again.
Would have been better to not commit this change in the first place, but if you squash the commits on merge it will have never existed in the master branch.

**Comment by tugberk92** [comment]
> > May I ask:
> > why Xerces.sh is being deleted,
> > why there is an blank line added to fairroot.sh
> > why the change in line 46/47 of fairship.sh
> > Thomas
> 
> He added xerces.sh (not xerces**c**.sh) in his first commit and then deletes it again.
> Would have been better to not commit this change in the first place, but if you squash the commits on merge it will have never existed in the master branch.

How would I do it(squash the commit on merge) now without renewing other commits or with an easy trick?

**Comment by olantwin** [comment]
Thomas can easily squash the commits when he merges them. If you wanted to do it, `git rebase -i` is the best way to do it. It's a very useful tool to rewrite history :)

**Comment by ThomasRuf** [comment]
could I suggest to cancel this request and Atakan starts again with just submitting what needs to be changed? I am confused with these many back and forth.

**Pull request closed without merging**